functions.lisp

```diff
@@ -11,6 +49,10 @@
     (<= 0 index (1- (length object))))
   (:method ((key string) (object hash-table))
     (nth-value 1 (gethash key object)))
+  (:method ((pointer pathname) object)
+    (if (equal #p"" pointer)
+        t
```

Comment:
What does it mean to return T here?

Reply:
It means that object has itself. Empty JSON Pointer is resolved to the object itself, whatever the object is. So, whatever is passed as OBJECT argument, `jhas` itself as `#p""`-indexed entry.